tests/test_blocks/test_step_impl/test_ek1_projmatfree_d_nu.py

```diff
@@ -0,0 +1,103 @@
+"""Tests for traditional EK1 implementations."""
+
+from functools import partial
+
+import jax
+import jax.numpy as jnp
+import pytest
+
+from tornadox.blocks.sde import ibm
+from tornadox.blocks.step_impl.first_order import ek1_projmatfree_d_nu
+
+
+@partial(jax.jit, static_argnames=("n", "d"))
+def _setup(*, n, d):
+    """Setup for derivative-dimension ordering."""
+
+    # Preconditioners
+    p, p_inv = ibm.preconditioner_diagonal(dt=0.1, num_derivatives=n)
+    p, p_inv = jnp.tile(p, reps=d), jnp.tile(p_inv, reps=d)
+
+    # System matrices
+    eye_d = jnp.eye(d)
+    a, q_sqrtm = ibm.system_matrices_1d(num_derivatives=n)
+    a, q_sqrtm = jnp.kron(eye_d, a), jnp.kron(eye_d, a)
+
+    # Projection matrices
+    e0, e1, *_ = jnp.eye(n + 1)[:, None, :]
+    e0, e1 = jnp.kron(eye_d, e0), jnp.kron(eye_d, e1)
```

Comment:
Similar spirit to the previous comment: How about a `iwp.projection_matrix(num_derivatives, num_dims, derivative_to_project_onto)` function or something like this?

Reply:
I think same answer as above: it would move a bunch of decisions to a lower level (for example the ordering), and the projection matrices are extremely rarely used only anyway. (In fact, looking at this now they are not used in this test or the rest of the code at all, so I better delete this bit)

Reply:
I would say that if we decide on an ordering, then it would make sense to provide projection matrices. But my motivation here is maybe even more about "using tornadox to quickly try out new things" than for what the package itself needs.